apps/mudita-center-e2e/src/specs/overview/kompakt-contacts-search.ts

```diff
@@ -0,0 +1,211 @@
+
+    // Verify that results 1 and 2 are displayed
+    const firstResult = await ContactsKompaktPage.getSearchSuggestionListResult(
+      0
+    ).isDisplayed()
+    const secondResult =
+      await ContactsKompaktPage.getSearchSuggestionListResult(1).isDisplayed()
+
+    expect(firstResult).toBe(true)
+    expect(secondResult).toBe(true)
+  })
+  it("Remove search phrase, input wrong search phrase (not found), check if no results is displayed", async () => {
+    //click remove/close button (X) in search field to remove searched phrase
+    const searchSuggestionsListEraseButton =
+      ContactsKompaktPage.searchSuggestionsListEraseButton
+    await expect(searchSuggestionsListEraseButton).toBeDisplayed()
+    await searchSuggestionsListEraseButton.click()
+
+    //input non-searchable phrase
+    const searchField = ContactsKompaktPage.searchField
+    await searchField.click()
+    await searchField.setValue("@#@#@#@#@@#hisdhisdh")
+
+    const searchSuggestionsListEmpty =
+      ContactsKompaktPage.searchSuggestionsListEmpty
+    await expect(searchSuggestionsListEmpty).toBeDisplayed()
+    const searchSuggestionsListEmptyText =
+      ContactsKompaktPage.searchSuggestionsListEmptyText
+    await expect(searchSuggestionsListEmptyText).toHaveText(
+      "We couldn't find anything..."
+    )
+  })
+
+  it("Test search Rules", async () => {
+    //1.0 check if search suggestions appear after inputing 1 character
+    const searchField = ContactsKompaktPage.searchField
+    await searchField.click()
+    await searchField.setValue("a")
+    const searchSuggestionsList = ContactsKompaktPage.searchSuggestionsList
+    await expect(searchSuggestionsList).toBeDisplayed()
+    //1.1 clear search field
+    const searchSuggestionsListEraseButton =
+      ContactsKompaktPage.searchSuggestionsListEraseButton
+    await expect(searchSuggestionsListEraseButton).toBeDisplayed()
+    await searchSuggestionsListEraseButton.click()
+
+    //2.0 input a number and check if there are many search results on the list
+    await searchField.setValue("1")
+    //2.1 check if there are more than 4 results
+    const results = await $$(
+      '//li[@data-testid and starts-with(@data-testid, "ui-form-search-results-")]'
+    )
+    await expect(results.length).toBeGreaterThan(4) //there are 5 results for this search
+    //2.2 clear search field
+    await searchSuggestionsListEraseButton.click()
+
+    //3.0 input an empty space " " and check if there are many search results on the list
+    await searchField.setValue(" ")
+    // 3.1 check if 17 results will appear
+    const updatedresults = await $$(
+      '//li[@data-testid and starts-with(@data-testid, "ui-form-search-results-")]'
+    )
+    await expect(updatedresults.length).toBeGreaterThan(16) //there are 17 results for this search
+    //3.2 clear search field
+    await searchSuggestionsListEraseButton.click()
+
+    //4.0 search using only phone number
+    const telNum: string = "+48345678902"
```

Comment:
```suggestion
    const telNum = "+48345678902"
```
This will be fine too as TypeScript will figure out the type from the value itself.

Same for website and email consts.